docs/js/features/odata/generate-client.md

```diff
@@ -0,0 +1,115 @@
+---
+id: generator-js-sdk
+title: Generate a typed OData client for JavaScript
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Generate an OData client for JavaScript
```

Comment:
Are the title and sidebar label intentionally different? They have almost the same length...